modules/react/text/lib/Text.tsx

```diff
@@ -56,6 +67,39 @@ const validateProps = ({typeLevel, variant, ...props}: TextProps) => {
   return updatedProps;
 };
 
+/**
+ * ## Text
+ * [View Docs](https://canvas.workday.com/components/text/text)
+ * 
+ * ---
+ * This is a generic base component intended to render any text.
+ * It's built on top of the `Box` component, so it has access to all [BoxProps](https://canvas.workday.com/components/layout/box/#props).
+ * It also has `typeLevel` and `variant` style props that simplify navigating our type hierarchy and using [Canvas type tokens](https://canvas.workday.com/tokens/type#type-styles).
+ * By default, it renders a semantic `span` element, but you can adjust this as needed with the `as` prop.
+ * 
+ * The type hierarchy is organized into four levels, which correspond to the type levels in Figma.
```

Comment:
Not a huge deal, but we may want to remove this reference to Figma given that canvas-kit is open source. We strip out all references to Figma on the external site.